src/components/SlateEditor/plugins/codeBlock/CodeBlock.jsx

```diff
@@ -98,36 +98,31 @@ class CodeBlock extends Component {
   }
 
   render() {
-    const { t, editor, node } = this.props;
-    const { editMode, showMenu } = this.state;
+    const { editor, node } = this.props;
+    const { editMode } = this.state;
     const { model } = getInfoFromNode(node);
-    const { top, left } = this.getMenuPosition();
+
+    const removeCodeblock = (
+      <Button stripped onClick={this.handleRemove}>
+        <Cross />
+      </Button>
+    );
 
     return (
       <>
-        <div
+        <CodeDiv
+          draggable
```

Comment:
Skal denne gjøre at man kan flytte blokken? For det virker ikke hos meg hvertfall

Reply:
Det var tanken, men tabIndex-propen forhindrer flytting. Om du tar vekk den lar den seg flytte, men det feiler når du slipper.